frontend/src/app/shared/components/footer/footer.component.scss

```
@@ -14,7 +19,12 @@
 }
 
 .footer-content {
+  @include r.tablet-up {
+    flex-direction: row;
+    gap: 24px;
+  }
   display: flex;
```

Stylelint (17.10.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)